AccountService.js

```diff
@@ -40,4 +40,71 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method getAll
+   * @param {Object} config
+   * @param {string} config.url - the base url
+   * @param {string} config.email - the user's email
+   * @param {string} config.after - an account's DID
+   * @param {number} config.limit - how many accounts to return
+   * @return {Array} data
+   * @description calls on the index route and returns all
+   * accounts that match the email passed in.
+   */
+  async getAll({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
+      params: {email, after, limit},
+      headers: {'Accept': 'application/ld+json, application/json'}});
+    return response.data;
+  }
+
+  /**
+   * @method update
+   * @param {Object} config
+   * @param {string} config.url
+   * @param {string} config.id - an account's id
+   * @param {number} config.sequence - an account's sequence number
+   * @param {Array<Object>} config.patch - an array of json patches
+   * @return {Void}
+   * @description updates an account via a series of json patches
+   * patches need to be in the:
+   * [json patch format]{@link https://tools.ietf.org/html/rfc6902}
+   * [we use fast-json]{@link https://www.npmjs.com/package/fast-json-patch}
+   * for handling json patches.
+   */
+  async update({url = this.config.urls.base, id, sequence, patch}) {
+    await axios.patch(`${url}/${id}`,
```

Comment:
one param indented, they should all be indented.  Same applies elsewhere.

Reply:
further explanation needed.

Reply:
Here's one of the bullets from here: https://github.com/digitalbazaar/bedrock/blob/master/CONTRIBUTING.md

Line break at 80 chars; if you must line break for function parameters, line break at the opening parenthesis and move all parameters down, do not break in the middle of the parameter list. The only exception to this is if a parameter is an object with too many properties to fit on a single line; you may break after the opening object bracket for this. Break before periods for chaining function calls.

Reply:
this branch is protected so I had to create another branch with the changes.

Reply:
lol so finally an explanation of that rule. I actually messed around with eslint last weekend with func-parens and max-length. check the upcoming PR I think I got it.